elyra/util/url.py

```diff
@@ -71,10 +71,10 @@ def get_verify_parm(default: bool = True) -> Union[bool, str]:
     """
     Returns a value for the 'verify' parameter of the requests.request
     method (https://requests.readthedocs.io/en/latest/api/). The value
-    is determined as follows: if environment variable CA_CERT_BUNDLE_PATH
+    is determined as follows: if environment variable TRUSTED_CA_BUNDLE_PATH
     is defined, use its value, otherwise return the default value.
     """
-    if os.environ.get("CA_CERT_BUNDLE_PATH"):
-        return os.environ.get("CA_CERT_BUNDLE_PATH")
+    if len(os.environ.get("TRUSTED_CA_BUNDLE_PATH", "").strip()) > 0:
```

Comment:
mmh, in test_url.py, you use the syntax to check for None, because that is what is.environ.get returns when the env is not defined.

https://github.com/elyra-ai/elyra/pull/2912/commits/52a5311b07c2eb009c79100b836e6f75c91ae6d9#diff-15011a921711725be34a8ee8be950816fa7e560f5aaf1c06df2dde61d7c0bedfL94

Why here length of a string instead?

Reply:
The purpose of the test in the pytest fixture and here is different. In the fixture the goal is to preserve the value of the environment variable, irrespective of whether it  defines a valid value or not. Therefore it's sufficient to check if the variable is defined and ignore its content. Here we must check for values that are known to cause issues and ignore them. (The empty string disables verification, a string containing only white spaces is not a valid filesystem reference.)

Reply:
Ok, that is a very good point, thanks. I see the idea now behind default value, if not defined, being the empty string.

Regarding string empty or not, if defined, we could even go further and do some type of regexp-matching.

Reply:
Yes, it'd be easy to check whether the specified string refers to an existing local file. In a nutshell it comes down to semantics (empty string/white spaces only = no intentional input vs string with errors). In my implementation I decided that we should differentiate between the two types of 'invalid' inputs.

If we were to silently ignore the variable if it contains an invalid file system reference (e.g. `/pth/to/file.pem` - which contains a typo - should have been `path`) the HTTP request would fail with a rather generic message:
```
requests.exceptions.SSLError: HTTPSConnectionPool(host='www.google.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))
```

If we don't check whether the file exists then the same HTTP request will fail with a message that highlights the problem:
```
OSError: Could not find a suitable TLS CA certificate bundle, invalid path: /pth/to/file.pem
```

Therefore if we were to check whether the file exists and raise a custom error we would not be adding any additional value over what we are already getting for free.